crates/rolldown/src/utils/tweak_ast_for_scanning.rs

```diff
@@ -62,7 +62,7 @@ fn split_top_level_variable_declaration<'a>(
         .iter()
         // TODO: support nested destructuring tree shake, `export const {a, b} = obj; export const
         // [a, b] = arr;`
-        .all(|declarator| matches!(declarator.id.kind, BindingPatternKind::BindingIdentifier(_)))
+        .any(|declarator| matches!(declarator.id.kind, BindingPatternKind::BindingIdentifier(_)))
```

Comment:
If there is no `BindingIdentifier` Declarator found, we don't need to split them at all, since they will not be tree shaken anyway. at least for now.